recipes/nionui-tool/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "nionui-tool" %}
+{% set version = "0.4.0" %}
+
+{% set pyver = "cp36.cp37.cp38" %}
+{% set pyabi = "abi3" %}  # [osx or linux]
+{% set pyabi = "none" %}  # [win]
+{% set platform = "macosx_10_11_intel" %}  # [osx]
+{% set platform = "manylinux1_x86_64" %}  # [linux]
+{% set platform = "win_amd64" %}  # [win]
+{% set fn = "nionui_tool-{}-{}-{}-{}.whl".format(version, pyver, pyabi, platform) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/nion-software/{{ name }}/releases/download/{{ version }}/{{ fn }}
+    sha256: 4d55734e882dc3a2b990a77e7a1e59f6693798c4a64163a333e1ee55df32e131  # [linux]
+    sha256: c392f6921b48ece3c0da80aaad0ec797771d16d1935b2854976cd987b50ecc2d  # [osx]
+    sha256: a94b96e6655cdb21402ba21c98f18b5cf62b5d041242d85473990c17e6ce24fd  # [win]
+    folder: wheels
```

Comment:
Please build from source, not from wheels

Reply:
No longer building from wheels.